docs/authentication/saml-at-clerk.mdx

```diff
@@ -1,3 +1,144 @@
 ---
-sanity_slug: /docs/authentication/saml-at-clerk
+title: SAML at Clerk (Beta)
+description: Clerk supports Enterprise SSO via the SAML protocol so that you can create authentication strategies for Identity Providers, such as Okta.
 ---
+
+# SAML at Clerk (Beta)
+
+Clerk supports Enterprise SSO via the SAML protocol so that you can create authentication strategies for Identity Providers, such as Okta.
+
+## Prerequisites
+
+- A Clerk application with the [Business](https://clerk.com/pricing) plan.
+- An `email_address` enabled as a strategy. To do so, navigate to the [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) section of the Clerk Dashboard. In the **Contact information** section, enable the **Email address** strategy by toggling on the switch.
+
+## Create a SAML connection
+
+To create a SAML Connection, get started by visiting the [Enterprise Connections](https://dashboard.clerk.com/last-active?path=user-authentication/enterprise-connections) section in the Clerk Dashboard.
+
+Click on the **Create connection** button.
+
+<Images
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/enterprise-connections.webp"
+  alt="The Enterprise Connections section in the Clerk Dashboard. There is a red arrow pointing to the 'Create connection' button."
+/>
+
+Once you have clicked on the **Create connection** button, you will be presented with a modal to create a new connection. Fill in the required fields.
+
+- **Name** - A unique name for the connection. This is so you can tell your connections apart.
+- **Domain** - This is the email domain for which you will enable Enterprise SSO.
+
+<Images
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/create-connection.webp"
+  alt="The Enterprise Connections section in the Clerk Dashboard. The 'Create connection' modal is open."
+/>
+
+Once you have filled in these fields, you will be directed to the configuration settings page for the connection you just created.
+
+<Images
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/enterprise-connection-settings-1.webp"
+  alt="The configuration settings for the Enterprise connection that was created."
+/>
+
+Scroll down to the **Service provider details** section. You will need to add these two fields to your IdP’s application:
+
+- SP Entity ID - This is a unique identifier for your SAML connection that your IdP application needs. Different providers use different terminology so consult our glossary table to locate the right term for your case.
+- ACS URL - This is your application’s URL that your IdP will redirect your users back to after they have authenticated in your IdP. Different providers use different terminology so consult our glossary table to locate the right term for your case.
```

Comment:
"Different providers use different terminology so consult our glossary table to locate the right term for your case."
Maybe move these to a single Callout below the bullets to bring more attention?

There should be a link to the Glossary below.